nat-lab/tests/test_wg_adapter.py

```diff
@@ -0,0 +1,80 @@
+import asyncio
+import platform
+from utils.connection_util import (
+    ConnectionTag,
+    get_libtelio_binary_path,
+    new_connection_raw,
+)
+from utils.output_notifier import OutputNotifier
+
+if platform.machine() != "x86_64":
+    import pure_wg as Key
+else:
+    from python_wireguard import Key  # type: ignore
+
+
+async def test_wg_adapter_cleanup():
+    output_notifier = OutputNotifier()
+
+    async def output_checker(stdout: str) -> None:
+        for line in stdout.splitlines():
+            print(line)
```

Comment:
This print could come with some extra context to make it easier to find if it's intentional to be here

Reply:
But it is not that hard to find that print, since this test doesn't come with all additional clutter from telio class, also only 1 node, and also not much is printed there, since we just start and stop tcli immediately

Reply:
If it's easy to find in the logs then it's fine :)